bot.py

```diff
@@ -63,6 +74,11 @@ def __init__(self):
         self._config = None
         self._db = None
 
+        log_level = logging._nameToLevel.get(self.config.log_level.upper())
+        if log_level is not None:
+            logger.setLevel(log_level)
+            ch.setLevel(log_level)
+
```

Comment:
Maybe print a message to say it has been set or it hasn't been set

Reply:
Added:

    bot.py - INFO: -------------------------
    bot.py - INFO: Logging level: FATAL
    bot.py - INFO: -------------------------

or 

    bot.py - INFO: -------------------------
    bot.py - INFO: Invalid logging level set.   [this line is red]
    bot.py - INFO: Using default logging level: INFO
    bot.py - INFO: -------------------------